template/package.json

```diff
@@ -11,7 +11,7 @@
     "unit": "jest --config test/unit/jest.conf.js --coverage",
     {{/if_eq}}
     {{#if_eq runner "karma"}}
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
+    "unit": "karma start test/unit/karma.conf.js --single-run",
```

Comment:
we set the NODE_ENV to `test` in the karma config, so BABEL_ENV will fall back to this.